### DISCUSSION TITLE
zero_order_tikhonov method?

### DISCUSSION THREAD

**Discussion by wkitlasten** [pypest/pyemu#405] [General]
Hi,

 Can someone point me to the `zero_order_tikhonov()` method. Seems like it should be a Pst class method, which seems to be defined in pyemu/pst/pst_handler.py. I'm not seeing anything there or in `pst_utils.py()`

**Comment by jtwhite79** [comment]
[https://github.com/pypest/pyemu/blob/7ff7445996b3a6bea1e825fac66e21d8f8bdc9dd/pyemu/utils/helpers.py#L938](https://github.com/pypest/pyemu/blob/7ff7445996b3a6bea1e825fac66e21d8f8bdc9dd/pyemu/utils/helpers.py#L938)

Its in helpers so it can be friends with first-order Tikhonov...
